tests/vite-plugin-pwa.ts

```diff
@@ -0,0 +1,25 @@
+import { runInRepo } from '../utils'
+import { RunOptions } from '../types'
+
+export async function test(options: RunOptions) {
+	if (options.viteMajor < 4) {
+		// only latest version
+		return
+	}
+
+	await runInRepo({
+		...options,
+		repo: 'vite-pwa/vite-plugin-pwa',
+		branch: 'userquin/feat-add-tests',
```

Comment:
this should be updated to your main branch before we merge this PR

Reply:
ok, I'll use shell-emulator and remove cross-env dependency...